packages/http/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@tech-matters/http",
```

Comment:
I'm concerned that there's an overloading of the term 'hrm' here. We are using it both to describe things relating to the HRM 'domain' - i.e. contact & case recording - currently the HRM API and the transcript processing lambdas, but also it being a catchall term to refer to the whole Aselo backend

In this case I deliberately didn't use 'hrm' in the name because I didn't want people assuming that the library was something exclusively used by the HRM domain, when it's shared between it and the resources domain (I know it's weird that we have 2 'domains' living in the same process, but I think it still works as an abstract boundary)

I suppose we could scope it to something broader like 'aselo-http' if you prefer? I think it's easy to change in the future though, since it's an internal module and updating the module name and all its dependents can be done atomically

Reply:
Ahh. There is likely a disconnect here and it is probably a question I should have asked a long time ago:

What does "HRM" actually mean in the context of aselo?

I've definitely just been thinking of it as a catch all for our "backend services". But defining what it means as a name and "bucket" now and what we want it to mean in the future is probably something we should all align on. ;)

Reply:
It stands for Helpline Relationship Management (like CRM). Which could mean a few things to be fair, but I see it as referring to a domain covering the recording & accessing of information about contacts (contacts & cases in our case), whereas I would like resources to be a separate domain outside of 'HRM'

It's only really coming up as a point now because until now 'The HRM domain' and 'The Aselo Backend' were pretty much synonymous